src/Model/GroundWaterTransport/gwt1.f90

```diff
@@ -124,77 +93,51 @@ subroutine gwt_cr(filename, id, modelname)
     ! -- Set memory path before allocation in memory manager can be done
     this%memoryPath = create_mem_path(modelname)
     !
-    call this%allocate_scalars(modelname)
+    call this%allocate_tsp_scalars(modelname)
+    call this%allocate_gwt_scalars(modelname)
     model => this
     call AddBaseModelToList(basemodellist, model)
     !
-    ! -- Assign values
-    this%filename = filename
-    this%name = modelname
-    this%macronym = 'GWT'
-    this%id = id
-    !
-    ! -- set input model namfile memory path
-    input_mempath = create_mem_path(modelname, 'NAM', idm_context)
-    !
-    ! -- copy option params from input context
-    call mem_set_value(lst_fname, 'LIST', input_mempath, found%list)
-    call mem_set_value(this%iprpak, 'PRINT_INPUT', input_mempath, &
-                       found%print_input)
-    call mem_set_value(this%iprflow, 'PRINT_FLOWS', input_mempath, &
-                       found%print_flows)
-    call mem_set_value(this%ipakcb, 'SAVE_FLOWS', input_mempath, found%save_flows)
-    !
-    ! -- create the list file
-    call this%create_lstfile(lst_fname, filename, found%list)
-    !
-    ! -- activate save_flows if found
-    if (found%save_flows) then
-      this%ipakcb = -1
-    end if
-    !
-    ! -- log set options
-    if (this%iout > 0) then
-      call this%log_namfile_options(found)
-    end if
-    !
-    ! -- Create utility objects
-    call budget_cr(this%budget, this%name)
+    ! -- Call parent class routine
+    call this%tsp_cr(filename, id, modelname, 'GWT', indis)
     !
-    ! -- create model packages
-    call this%create_packages()
+    ! -- Create model packages
+    call this%create_gwt_specific_packages(indis)
```

Comment:
@langevin-usgs, yes, the name can stay the same. I'd like to mention that, gwt, gwe, and tsp would all have a `create_packages()` routine, and I was thinking for readability it would be nice to differentiate between the `create_packages()` when calling the routine for the 'elevated' packages (`ADV`, `FMI`, etc.) vs when calling the routine for GWT/GWE specific packages (`DSP`, `MST`, etc.).  I could either (1) reduce the routine name from `create_gwt_specific_packages()` to just `create_gwt_packages()` or just (2) have all 3 instances use `create_packages`, depending on your preference?

Reply:
I tried reverting back to simply `create_packages()`; however, this wouldn't compile in gfortran, likely because there is already an identically named function in tsp.f90, from which gwt.f90 inherits.  Thus, the answer to the original review comment is that yes, I do think this name had to change.  For documentation purposes, 7e8644d attempted to change the name in response to this comment, but since that broke the CI, reverted back in df5ea45.

Reply:
Implemented the RCH approach in `gwt1.f90` with f686ee1